src/test/java/com/ea/restaurant/utils/MockOauth2Util.java

```diff
@@ -0,0 +1,37 @@
+package com.ea.restaurant.utils;
+
+import com.ea.restaurant.constants.Oauth2;
+import com.ea.restaurant.entities.AppClient;
+import com.ea.restaurant.entities.AppClientScope;
+import com.ea.restaurant.utils.fixtures.AppAccessTokenFixture;
+import com.ea.restaurant.utils.fixtures.AppRefreshTokenFixture;
+import com.ea.restaurant.utils.fixtures.Oauth2Fixture;
+import com.nimbusds.jose.JOSEException;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+public class MockOauth2Util {
+
```

Comment:
You don't need to create a new file, I think we have done similar things in another file

Reply:
This is the first file that i create to prepare mocks for Oauth2Util